internal/provider/provider.go

```diff
@@ -148,14 +173,20 @@ func getProjectFromCredentials(ctx context.Context, client *clients.Client) (*mo
 	listProjParams.ScopeType = &scopeType
 	listProjResp, err := clients.RetryProjectServiceList(client, listProjParams)
 	if err != nil {
-		return nil, fmt.Errorf("unable to fetch project id: %v", err)
+		diags = append(diags, diag.Errorf("unable to fetch project id: %v", err)...)
+		return nil, diags
 	}
 	if len(listProjResp.Payload.Projects) > 1 {
-		return nil, fmt.Errorf("this version of the provider does not support multiple projects, upgrade to a later provider version and set a project ID on the provider/resources")
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Warning,
+			Summary:  "There is more than one project associated with the organization of the configured credentials.",
+			Detail: `The oldest project has been selected as the default. To configure which project is used as default, 
+			set a project in the HCP provider config block. Resources may also be configured with different projects.`,
+		})
+		return getOldestProject(listProjResp.Payload.Projects), diags
 	}
-
-	project := listProjResp.Payload.Projects[0]
-	return project, nil
+	project = listProjResp.Payload.Projects[0]
+	return project, diags
```

Comment:
Do we need here to still grab the 0th index of returned projects from the payload if existing projects are only length 1? Or would it make more sense to scrap those two lines and always use the return from our getOldestProject call?

Reply:
I'd say yes, mainly so we only throw the warning when more than one project is found. Maybe for the provider developer's context, throw in a comment about defaulting to the single project if only one is found.